### PR TITLE
Fs3

### PR DESCRIPTION
**Pull request by dbarbi1**
Updated Freedom Studio Projects to match beta 3 release

### CODE REVIEW

File: FreedomStudio/E31FPGA/coreplexip_welcome/.project [hunk 1]
```diff
@@ -114,6 +119,26 @@
 			<type>2</type>
 			<locationURI>virtual:/virtual</locationURI>
 		</link>
+		<link>
+			<name>bsp/libwrap/.DS_Store</name>
+			<type>1</type>
+			<locationURI>PARENT-3-PROJECT_LOC/bsp/libwrap/.DS_Store</locationURI>
```
Comment (ilg-ul) [inline FreedomStudio/E31FPGA/coreplexip_welcome/.project]: Are these links to `.DS_Store` necessay?

File: FreedomStudio/E31FPGA/dhrystone/sifive-coreplexip-e31-arty.cfg [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# JTAG adapter setup
```
Comment (ilg-ul) [inline FreedomStudio/E31FPGA/dhrystone/sifive-coreplexip-e31-arty.cfg]: The new version of OpenOCD no longer uses this file, functionality was split between interface and board.

File: FreedomStudio/E31FPGA/coreplexip_welcome/.project (hunk 1 above)
Comment (dbarbi1) [reply FreedomStudio/E31FPGA/coreplexip_welcome/.project]: Good catch, I will remove.

File: FreedomStudio/E31FPGA/dhrystone/sifive-coreplexip-e31-arty.cfg (hunk 2 above)
Comment (dbarbi1) [reply FreedomStudio/E31FPGA/dhrystone/sifive-coreplexip-e31-arty.cfg]: Sure, we will update to the config files included in the openocd builds in the near future.

**Pull request merged**